cmd/sourced/cmd/logs.go

```diff
@@ -0,0 +1,34 @@
+package cmd
+
+import (
+	"context"
+
+	"github.com/src-d/sourced-ce/cmd/sourced/compose"
+)
+
+type logsCmd struct {
+	Command `name:"logs" short-description:"Fetch the logs of source{d}" long-description:"Fetch the logs of source{d}"`
```

Comment:
"of source{d}" sounds weird to me. I would use "Fetch the source{d} logs" or "Fetch the logs of source{d} components"